pkg/pillar/cmd/diag/diag.go

```diff
@@ -880,7 +880,7 @@ func printOutput(ctx *diagContext, caller string) {
 		// fields are set and Dhcp type; proxy info order
 		ifname := port.IfName
 		isMgmt := types.IsMgmtPort(*ctx.DeviceNetworkStatus, ifname)
-		priority := types.GetPortCost(*ctx.DeviceNetworkStatus,
+		cost := types.GetPortCost(*ctx.DeviceNetworkStatus,
```

Comment:
I didn't know we were using "priority" in some cases. Thanks for making it consistently use "cost" and not mixing the two.